build/Dockerfile

```diff
@@ -130,10 +217,26 @@ RUN --mount=type=cache,target=/root/.cache/pip \
     # Cleanup the bdist whl file
     rm $(head bdist_name) /tmp/bdist_name
 
+## Final image ################################################
+FROM base as release
+ARG PYTHON_VERSION
+ARG SITE_PACKAGES=/opt/tuning/lib/python${PYTHON_VERSION}/site-packages
+ARG USER=tuning
+ARG USER_UID=1000
+
+# Copy in the full python environment
+COPY --from=python-installations /opt/tuning /opt/tuning
+
+ENV PATH=/opt/tuning/bin:$PATH
+
+# Print a list of all installed packages and versions
+RUN pip list -v --disable-pip-version-check --no-python-version-warning
+
+ENV HOME=/home/tuning
```

Comment:
Does this mean the working directory is `/home/tuning`? Should the scripts be copied into this directory instead of `/app`? And what are the implications, I know when I've run before I've hit into issue with creating the `.aim` directory and the output directory in `/tmp`. Since you removed the ownership of those dirs to not include the new user, are those directories still able to be created?